src/main/java/uk/gov/ons/ctp/response/collection/exercise/domain/SampleLink.java

```diff
@@ -39,4 +42,7 @@
   @Column(name = "samplesummaryid")
   private UUID sampleSummaryId;
 
+  @Enumerated(EnumType.STRING)
+  @Column(name = "statefk")
+  private LinkSampleSummaryDTO.SampleLinkState state;
```

Comment:
I'm not quite sure why this is imported as LinkSampleSummaryDTO.SampleLinkState rather than just SampleLinkState (this goes for all other instances of this happening in this PR)

Reply:
Don't know either - IDE insisted on importing the root class and framing all the enum references like that. I've manually changed it now